tsconfig.json

```diff
@@ -4,7 +4,9 @@
     "baseUrl": ".",
     "paths": {
       "@stoplight/spectral-core": ["src/index.ts"],
-      "@stoplight/spectral-functions": ["src/functions/index.ts"]
+      "@stoplight/spectral-functions": ["src/functions/index.ts"],
+      "@stoplight/spectral-formats": ["src/formats/index.ts"],
```

Comment:
more packages exposed.
These paths won't be needed as soon as we have monorepo.